sdk/src/main/scala/com/horizen/fork/ForkConfigurator.scala

```diff
@@ -0,0 +1,21 @@
+package com.horizen.fork
+
+import scala.util.{Failure, Success, Try}
+
+abstract class ForkConfigurator {
+  def getBaseSidechainConsensusEpochNumbers():scConsensusEpochNumber
```

Comment:
I think that the `BaseFork` must always refer to all `0` epoch numbers.
So we can set this method as final returning the `scConsensusEpochNumber(0,0,0)`.
For example, OpenStakeTx and BT limit are going to be a part of `SidechainFork1`